Libraries/Components/ProgressBarAndroid/__tests__/ProgressBarAndroid-test.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ * @emails oncall+react_native
+ * @flow
+ */
+
+'use strict';
+
+const React = require('React');
+// $FlowFixMe
+const ProgressBarAndroid = require('../ProgressBarAndroid.android');
```

Comment:
@cpojer do you know how to fix this?

Reply:
This is fine. The iOS flow config doesn't see these modules. I don't think it matters much.